aiozipkin/span.py

```diff
@@ -117,6 +121,9 @@ def kind(self, span_kind: str) -> 'NoopSpan':
     def name(self, span_name: str) -> 'NoopSpan':
         return self
 
+    def new_child(self, name: str = None, kind: str = None) -> 'NoopSpan':
+        return self
```

Comment:
to be on safe side lets create new NoopSpan here

Reply:
done... maybe